luda-editor/luda-editor-client/src/app/editor/timeline/timeline_body/track_body/subtitle_track_body/subtitle_clip_body.rs

```diff
@@ -1,6 +1,8 @@
+use std::f32::consts::PI;
+
 use crate::app::{
     editor::{events::EditorEvent, TimelineRenderContext},
-    types::{PixelSize, SubtitleClip, Time},
+    types::{PixelSize, SubtitleClip, Time, TimePerPixel},
```

Comment:
```suggestion
    types::{PixelSize, SubtitleClip, Time},
```
`TimePerPixel` is unused. We can remove this